tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/runtime/CustomRuntimeOSGiTest.java

```diff
@@ -57,19 +58,16 @@
 
     @Configuration
     public Option[] createConfiguration() {
-
-        List<Option> optionList = new ArrayList<>();
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.runtime.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.runtime.mss").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.runtime.jar").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
-                .versionAsInProject());
-
-        optionList = OSGiTestConfigurationUtils.getConfiguration(optionList, null);
-        return optionList.toArray(new Option[optionList.size()]);
+        return new Option[] {
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.runtime.mgt").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.runtime.mss").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.runtime.jar").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
```

Comment:
All the osgi tests are following the same pattern.
[1] https://github.com/wso2/carbon-datasources/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/datasource/osgi/DataSourceManagementServiceTest.java

[2] https://github.com/wso2/carbon-jndi/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/jndi/osgi/JNDITest.java